client/checkout/api/index.js

```diff
@@ -377,6 +377,8 @@ export default class WCPayAPI {
 			// eslint-disable-next-line camelcase
 			save_payment_method: savePaymentMethod,
 			action: 'update_payment_intent',
+			// eslint-disable-next-line camelcase
+			_ajax_nonce: getConfig( 'updatePaymentIntentNonce' ),
```

Comment:
This isn't really related to the purpose of this PR...but why shouldn't we be sending a nonce with this AJAX request like the other AJAX requests we make?